runtime/lang/menu_eo.utf-8.vim

```diff
@@ -459,5 +459,12 @@ menutrans Co&lor\ test				Testo\ de\ &koloroj
 menutrans &Highlight\ test			Testo\ de\ &emfazo
 menutrans &Convert\ to\ HTML			Konverti\ al\ &HTML
 
+menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	Baskuligi\ relativa\ linio\ numerado<Tab>:set\ nru!
+menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		Baskuligi\ linio\ envolvado<Tab>:set\ wrap!
+menutrans Toggle\ W&rapping\ at\ word<Tab>:set\ lbr!		Baskuligi\ envolvante\ ĉe\ vorto<Tab>:set\ lbr!
+menutrans Toggle\ Tab\ &Expanding<Tab>:set\ et!			Baskuligi\ langeto\ ekspansiiĝanta<Tab>:set\ et!
+menutrans Toggle\ &Auto\ Indenting<Tab>:set\ ai!		Baskuligi\ aŭtomata\ indentado<Tab>:set\ ai!
```

Comment:
For consistency with terminology used in `eo.po` and with https://komputeko.net/#indent  I suggest:

```
Baskuligi\ aŭtomatan\ krommarĝenon<Tab>:set\ ai!
```

At least the accusative is missing in the PR.